install_system_deps.sh

```diff
@@ -1,21 +1,49 @@
 #!/bin/bash
 
-OS=$1
 
-if [ "$OS" = "Linux" ]; then
-    echo "Installing GeoDjango dependencies for $OS"
+# This script installs the necessary libs for GeoDjango and should be ran from the repository root.
+# It detects which OS you're using and proceeds with the correct installation method.
+# This works for any version of Linux or OSx.
+# Make sure you run "chmod +x ./install_system_deps.sh" to make it executable.
+
+# Colors for colored output
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+YELLOW='\033[0;33m'
+BLUE='\033[0;34m'
+CYAN='\033[0;36m'
+RESET='\033[0m'
+
+
+# Start script
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
```

Comment:
This definitely breaks the script for me, just fyi.  My OS comes thru as `Linux`.  No worries, I'll probably not ever use this script again lol

Reply:
Do this, I'm positive it would work.

<img width="253" alt="Screen Shot 2021-02-21 at 7 59 38 PM" src="https://user-images.githubusercontent.com/29584063/108645057-71921600-747f-11eb-86fc-45044df12948.png">

Reply:
using uname yes you'll get` Linux`, using `OSTYPE` you'll get `linux-gnu`

Reply:
Woah, you're right!  Why did that never come up when I was searching for "Detecting OS in bash" lol